app/client/src/mocks/handler.ts

```diff
@@ -0,0 +1,44 @@
+import {  rest } from 'msw'
```

Comment:
Not important but if you have a linter it didnt seem to catch the space here.

Reply:
so strict